src/components/pinning-manager/pinning-manager-modal/PinningManagerModal.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import i18n, { localesList } from '../../../i18n'
+import { getLanguage } from '../../../lib/i18n'
+
+// Components
+import { Modal, ModalBody, ModalActions } from '../../modal/Modal'
+import SpeakerIcon from '../../../icons/StrokeSpeaker'
+import Button from '../../button/Button'
+
+const LanguageModal = ({ t, tReady, onLeave, link, className, ...props }) => {
+  const handleClick = (lang) => {
+    i18n.changeLanguage(lang)
+    onLeave()
+  }
+
+  return (
+    <Modal {...props} className={className} onCancel={onLeave} style={{ maxWidth: '40em' }}>
```

Comment:
This modal is just a placeholder (copied from language modal), will also update it next :)